### ISSUE TITLE
Docs: indent_size wrong possible_values?

### ISSUE THREAD

**Issue by Lilja** [editorconfig/editorconfig#469]
This might be petty, but in the wiki page here that is linked from the editor config website, the `indent_size` declarations's possible values is considered `integer`, compare that to `tab_width` which says it can only be a positive integer.

Shouldn't `indent_size` also only accept a positive integer?

**Comment by xuhdev** [comment]
Yes, both must be positive. It's not petty at all; actually, our formal spec also missed this point: https://editorconfig-specification.readthedocs.io/#supported-pairs
